socorro/unittest/processor/test_breakpad_transform_rules.py

```diff
@@ -310,41 +297,34 @@ def test_stackwalker_fails_2(self, mocked_subprocess_module):
         assert processed_crash.mdsw_status_string == "unknown error"
         assert not processed_crash.success
         expected = [
-            "MDSW output failed in json: Expected String or Unicode",
-            "MDSW failed on 'upload_file_minidump': unknown error"
+            '/bin/stackwalker output failed in json: Expected String or Unicode',
+            (
+                'MDSW failed on \'timeout -s KILL 30 /bin/stackwalker --raw-json '
+                '00000000-0000-0000-0000-000002140504.json --symbols-url https://localhost '
+                '--symbols-url https://localhost --symbols-cache /mnt/socorro/symbols '
+                'a_fake_dump.dump 2>/dev/null\': unknown error'
+            )
```

Comment:
`BreakpadStackwalkerRule2015` produces slightly different output. So this just updates it to the new output.